src/test/java/net/snowflake/client/jdbc/SnowflakeTimestampNTZAsUTCTest.java

```diff
@@ -74,8 +79,9 @@ public static void restoreTimeZone() {
   public void testTimestampNTZ() throws Throwable {
     TimeZone.setDefault(TimeZone.getTimeZone(timeZone));
     LocalDateTime dt = parseTimestampNTZ(this.inputTimestamp);
-    SnowflakeTimestampNTZAsUTC stn =
-        new SnowflakeTimestampNTZAsUTC(dt.toEpochSecond(ZoneOffset.UTC) * 1000, dt.getNano());
+    SnowflakeTimestampWithSessionTimezone stn =
+        new SnowflakeTimestampWithSessionTimezone(
+            dt.toEpochSecond(ZoneOffset.UTC) * 1000, dt.getNano(), TimeZone.getTimeZone("UTC"));
```

Comment:
Looks like the name of this test file can be changed as well?

Reply:
Good point! Changed to SnowflakeTiemstampWithTimezoneTest.